Unique key update

fred-shaykis opened the pull request:
* suggestion to use MD5 to create a unique key rather than specifying multiple columns

fred-shaykis committed:
update readme file

fred-shaykis committed:
add md5 script

shaunymca commented:
Hey @fred-shaykis Thanks for working on this. I think the md5 script is very handy. Could you add a section in the readme on it's use case along with how to actually use it? I assume the user would create a column, and use it there, but could you elaborate it? There are a lot of non-technical users that set these spreadsheets up.

fred-shaykis committed:
added info on md5

fred-shaykis commented:
Hi @shaunymca  - let me know if what I've added in the last commit is what you were thinking of. I've added info on how to use it in sheets and linked to an article on Fishtown's blog that explains our most common use cases for md5.
